README.md

```diff
@@ -62,7 +62,16 @@ To run locally **Sugarizer Application into the Web Browser** (GNU Linux/Mac OS/
 
 For **Chrome**, close ALL running instances of Chrome and re-launch it using the command line:
 
- 	chrome --allow-file-access-from-files  index.html
+ 	chrome --allow-file-access-from-files file:\\\Directory:\sugarizer\index.html
+
+In windows, \ is the way path is written and / is the way for Linux/OSX backward and forward slashes differ here, hence path for Linux/MacOS will be:
+
+	chrome --allow-file-access-from-files file:///Directory:/sugarizer/index.html
+
+Directory in the path above is where you have stored/cloned your sugarizer repo into. Notice the path in the image below to get the gist of it.
```

Comment:
Write `<Directory>` instead. This paragraph should be before the one mentioning difference between Windows and Linux/MacOS